application/src/main/java/dev/ikm/komet/app/App.java

```diff
@@ -810,30 +798,27 @@ private void launchClassicKomet() throws IOException, BackingStoreException {
         appPreferences.sync();
         if (createJournalViewMenuItem != null) {
             createJournalViewMenuItem.setDisable(false);
-            KeyCombination newJournalKeyCombo = new KeyCodeCombination(KeyCode.J, KeyCombination.SHORTCUT_DOWN);
+                KeyCombination newJournalKeyCombo = new KeyCodeCombination(KeyCode.J, KeyCombination.SHORTCUT_DOWN);
             createJournalViewMenuItem.setAccelerator(newJournalKeyCombo);
             KometPreferences journalPreferences = appPreferences.node(JOURNAL_WINDOW);
         }
     }
 
-    public void openDatasetPage(){
+    private void openExport() {
         KometPreferences appPreferences = KometPreferencesImpl.getConfigurationRootPreferences();
         KometPreferences windowPreferences = appPreferences.node(MAIN_KOMET_WINDOW);
-
         WindowSettings windowSettings = new WindowSettings(windowPreferences);
-        Stage datasetStage = new Stage();
-        FXMLLoader datasetPageLoader = ExportDatasetViewFactory.createFXMLLoaderForExportDataset();
-        try {
-            Pane datasetBorderPane = datasetPageLoader.load();
-            ExportDatasetController datasetPageController = datasetPageLoader.getController();
-            datasetPageController.setViewProperties(windowSettings.getView().makeOverridableViewProperties());
-            Scene sourceScene = new Scene(datasetBorderPane, 550, 700);
-            datasetStage.setScene(sourceScene);
-            datasetStage.setTitle("Export Dataset");
-            datasetStage.show();
-        } catch (IOException e) {
-            throw new RuntimeException(e);
-        }
+        Stage exportStage = new Stage();
+        //set up ExportViewModel
```

Comment:
Can we make the Stage window modal?
```java
exportStage.initOwner(parentStage);
exportStage.initModality(Modality.APPLICATION_MODAL); 
...// rest of code

exportStage.showAndWait(); // move or replace this at line 821 
```

Reply:
Actually, we can approve and create another ticket to resolve this scenario "Modality".